README.md

```diff
@@ -64,33 +65,21 @@ $ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:${PKG_CONFIG_PATH}
 
 Then proceed to follow the build instructions below.
 
+#### Ubuntu/Debian
 
-### Ubuntu 12.x Prerequisites
-
-On recent Ubuntu versions, simply install all prerequisites as follows:
+On Ubuntu/Debian distributions, simply install all prerequisites as follows:
 
 ```
 # apt-get install build-essential autoconf automake libpcre3-dev libevent-dev pkg-config zlib1g-dev
```

Comment:
Consider using `apt` instead

Reply:
I may be wrong, but I think `apt-get` is less likely to be missing than `apt`.

Reply:
I concur